lib/ReactViews/Map/Panels/SharePanel/share-panel.scss

```diff
@@ -186,3 +187,15 @@
     box-sizing: content-box;
   }
 }
+
+.getShareSaveHelpText {
+  text-decoration: underline;
+  padding-top: 0.8em;
+  cursor: help;
+}
+
+hr.thin-line-divider {
+  border: 0;
+  height: 0;
+  border-top: 0.5px solid rgba(255, 255, 255, 0.15);
+}
```

Comment:
Normally I would say, we should refrain from writing new `scss` code, in this case though it might make sense to revisit it a later point and rewrite it using `tsx` and `styled-components`.

Reply:
OK Im getting my head around styled components Ill raise a ticket to do this in next sprint